build.gradle.kts

```diff
@@ -0,0 +1,74 @@
+group = "io.intrepid"
+version = "2.0.0-alpha1"
+
+plugins {
+    java
+    kotlin("jvm") version "1.3.11"
+    `kotlin-dsl`
+    id("com.gradle.plugin-publish") version "0.9.7"
+    `maven-publish`
+    `java-gradle-plugin`
+}
+
+buildscript {
+    repositories {
+        mavenCentral()
+        maven(url = "https://plugins.gradle.org/m2/")
+    }
+    dependencies {
+        classpath("org.gradle.api.plugins:gradle-nexus-plugin:0.3")
+    }
+}
+
+repositories {
+    google()
+    jcenter()
+    mavenLocal()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+}
+
+dependencies {
+    implementation(gradleApi())
+    implementation("com.android.tools.build:gradle:3.3.0")
```

Comment:
Can you update to the latest version of the Android Gradle Plugin while you're at it?

Reply:
Or, perhaps it would be best to get this PR merged in and then start a new one with all of the latest updates (Gradle 6, replacing FindBugs with SpotBugs, etc.)

Reply:
gonna keep it as 3.3 in this pr. Updating to the very latest android gradle version isn't necessary a good idea as it will force the plugin consumer to also update that version or newer